.github/workflows/publish-new-github-release.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish new github release
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ 'master-v2' ]
+    types:
+      - closed
+
+jobs:
+  release:
+    name: Publish new release
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/heads/master-v2') || ((startsWith(github.event.pull_request.head.ref, 'release/') && github.event.pull_request.merged == true) # only merged pull requests must trigger this job
```

Comment:
If I have understood, you made the new packages in v2 version branches. Here you are still triggering on release that is a v1 version right?

Reply:
also what about hotfix branches?

Reply:
This will be triggered only for PRs opened to master-v2. Added hotfix.